e2e/tests/cohorts/cohort-group-actions.spec.ts

```diff
@@ -186,9 +186,9 @@ describe('Build cohort page actions', () => {
     expect((await group1.findGroupCriteriaList()).length).toBe(2);
     totalCount = await cohortBuildPage.getTotalCount(); // New Total Count by 2 criteria.
 
-    // Rename Criteria: rename "Whole Genome Variant" to "NDA Sets".
+    // Rename Criteria: rename "Whole Genome Sequence" to "NDA Sets".
     const newCriteriaName = 'DNA Sets';
-    await group1.editCriteriaName('Whole Genome Variant', newCriteriaName);
+    await group1.editCriteriaName('Whole Genome Sequence', newCriteriaName);
```

Comment:
This will still need a change in DB right?

Reply:
Yes. I think the menu change in `synth_r_2019q4_17` and this change _should_ get all tests passing again.